### PR TITLE
Don't specify exact approver assignment command in approval-required …

### PR DESCRIPTION
**Pull request by BenTheElder**
…comment

Far too many people immediately copy+paste this command before any review occurs. The usage can still be found in the linked "[Kubernetes Code Review Process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)" document.

Also adds a missing space.

/hold
Holding because this needs review from various parties.

See: https://github.com/kubernetes/test-infra/issues/25806#issuecomment-1175285125 + in the wild actual review processes happening.

### CODE REVIEW

**Comment by nikhita** [comment]
It might still be a little confusing on how to assign someone for new contributors...but it might be worth trying this because the original text triggers github notifications for a large set of maintainers.

With contribex hat on - I'm okay with trying this text for a while. If we end up with lots of new contributors, we can revert. LGTM after tests pass.

@cblecker thoughts?

**Comment by BenTheElder** [comment]
> It might still be a little confusing on how to assign someone for new contributors...but it might be worth trying this because the original text triggers github notifications for a large set of maintainers.

I think the linked review flow document could do with some improving, perhaps something to do before merging something like this.

That said, I also think it's *good* that people need read about how approval works before knowing how to assign approvers, and reviewer can assign approvers after LGTM.

Right now we have an immense bias towards immediately assigning approvers + reviewers making the reviewers category essentially defunct overloading approvers. I nearly always see people comment with the bot suggested command as soon as it suggests it, even with the tweaked text and bolding. And frequently on low-quality PRs that were never going to be lgtm let alone approved.

**Comment by BenTheElder** [comment]
This originally came from https://github.com/kubernetes/test-infra/issues/25806#issuecomment-1116995142

**Comment by BenTheElder** [comment]
cc @jberkus @mrbobbytables

**Comment by cblecker** [comment]
/lgtm
/approve
/hold

Feel free to remove hold when you're ready for this to merge.

**Review by nikhita — APPROVED** [review]
/lgtm
/approve

:shipit:

**Review by MadhavJivrajani — APPROVED** [review]
/lgtm

**Comment by BenTheElder** [comment]
PSA sent
/hold cancel
/test all

**Pull request merged**
